.github/workflows/pr-auto-merge.yml

```diff
@@ -0,0 +1,33 @@
+on:
+  pull_request:
+
+jobs:
+  e2e:
+    uses: ./.github/workflows/_e2e.yaml
```

Comment:
```suggestion
  e2e:
    if: ${{ contains(github.event.pull_request.user.login, '["dependabot","github-actions"]') }}
    uses: ./.github/workflows/_e2e.yaml
```

Reply:
If we applied this change, I think the e2e job will not be executed when PR is created.

We needed to execute e2e job when PR is created regardless the pull request creator, thats why I didn't put this checking on this workflow.